libcxx/test/std/utilities/variant/variant.visit.member/robust_against_adl.pass.cpp

```diff
@@ -8,7 +8,6 @@
 
 // UNSUPPORTED: c++03, c++11, c++14, c++17, c++20, c++23
 // The tested functionality needs deducing this.
-// UNSUPPORTED: clang-17
```

Comment:
Here you also need to remove the comment above (here and elsewhere) if I recall correctly that's what the UNSUPPORTED was for (that was done by me but I don't remember very well). Please verify if that makes sense.

Reply:
@robincaloudis Also before you start dropping clang-17, AFAIK AIX, FreeBSD, etc. bots should be updated.